irb.gemspec

```diff
@@ -42,4 +42,5 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "reline", ">= 0.4.2"
   spec.add_dependency "rdoc", ">= 4.0.0"
+  spec.add_dependency "pp", ">= 0.6.0"
```

Comment:
Do we need to specify this instead of adding it just to Gemfile for testing?
User can install bug fixed version of pp if they want (but I know user usually don't update pp nor add pp to Gemfile)
It is beneficial, but adding it is not a usual case. We don't usually bump reline or rdoc requirements for each bug fixes.
What do you think?

Reply:
We need to specify it otherwise with the workaround removed, users with newer IRB but older pp will get a regression when dealing with basic objects.

> It is beneficial, but adding it is not a usual case. We don't usually bump reline or rdoc requirements for each bug fixes.

We do when we have to make corresponding changes on IRB side?

It's like when `repl_type_completor` adopts new Prism APIs/structures, it bumps its requirement for `prism`.

We can of course make the patch conditional based on the version of `pp` it uses, but IMO with the size of the current patch it's not worth it.